src/video/playVideoFrames.m

```diff
@@ -0,0 +1,28 @@
+function playVideoFrames(cfg)
```

Comment:
I think we could make this more general.

Maybe we could pass the info about a given trial by using a different structure than `cfg`.

For example when drawing `dots` for RDK we do thing like this:

```
dots = reseedDots(dots, cfg)
```

Because the way things are assumes that all trials have the same number of frames, the same video name...

Having a separate structure should also help save the output in a BIDS tsv using CPP_BIDS.